src/test/java/core/FormValidationTest.java

```diff
@@ -72,7 +73,7 @@ private void jsValidation() {
     }
 
     private void navigateAway() {
-        jenkins.runThenConfirmAlert(() -> jenkins.open());
+        jenkins.runThenConfirmAlert(() -> driver.findElement(By.xpath("//ol[@id=\"breadcrumbs\"]/li[1]/a")).click());
```

Comment:
Opening a new page suppresses the alert, but we can get the alert by clicking on the first breadcrumb, which accomplishes the same purposes.